esp/scm/ws_espcontrol.ecm

```diff
@@ -93,13 +93,59 @@ ESPresponse [exceptions_inline, nil_remove, http_encode(0)] SetSessionTimeoutRes
     string Message;
 };
 
-ESPservice [auth_feature("NONE"), version("1.01"), default_client_version("1.01"), exceptions_inline("./smc_xslt/exceptions.xslt")] WSESPControl
+ESPrequest [nil_remove] UnsubscribeFromDaliRequest
+{
+};
+
+ESPresponse [exceptions_inline, nil_remove, http_encode(0)] UnsubscribeFromDaliResponse
+{
+    int Status;
+    string Message;
+};
+
+ESPrequest [nil_remove] ReSubscribeToDaliRequest
```

Comment:
I'm not sure I like the "Re" part of the name.  I.e. you "Subscribe", "Unsubscribe", and then "Subscribe" again.